packages/components/src/utils/SecondsCountDown/SecondsCountDown.test.ts

```diff
@@ -0,0 +1,160 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest';
+
+import { SecondsCountDown } from './SecondsCountDown';
+
+describe('SecondsCountDown tests', () => {
+  beforeEach(() => {
+    vi.useFakeTimers({ toFake: ['Date', 'setTimeout'] });
+    vi.setSystemTime('2022-02-10T00:00:00.000Z');
+  });
+
+  afterEach(() => {
+    vi.restoreAllMocks();
+  });
+
+  it('Значение таймера при создании равно разнице текущей даты и указанной даты', () => {
+    const sut = new SecondsCountDown({
+      targetDate: new Date('2022-02-10T00:00:50.000Z'),
+    });
+
+    expect(sut.textTime).toBe('00:50');
+  });
+
+  it('Значение таймера при создании равно указанным секундам', () => {
+    const sut = new SecondsCountDown({
+      seconds: 50,
+    });
+
+    expect(sut.textTime).toBe('00:50');
+  });
+
+  it('Значение seconds игнорируется и значение таймера при создании равно разнице текущей даты и указанной даты, и значение seconds игнорируется', () => {
```

Comment:
```suggestion
  it('Значение seconds игнорируется при наличии targetDate', () => {
```

Может так?
Текущее название до конца не понял, очень много 'и'